src/api/app/controllers/webui/project_controller.rb

```diff
@@ -93,9 +93,31 @@ def subprojects
   end
 
   def pulse
+    switch_to_webui2
     @pulse = @project.project_log_entries.page(params[:page])
   end
 
+  def update_pulse
```

Comment:
The name is misleading. This is actually a `show`.

Reply:
Not really, `pulse` is the show. You could incorporate this into the same action (@ChrisBr already suggested it) but I haven't found a clever way to do it with `respond_to` yet...

Reply:
I try to find better names when I move this into it's own controller...